stable/deployment/values.yaml

```diff
@@ -54,13 +54,13 @@ replicaCount: 1
 commonLabels: {}
 ## @param commonAnnotations Annotations to add to all deployed objects
 ##
-commonAnnotations: {}
+commonAnnotations: { foo: bar }
```

Comment:
Please remove

Reply:
There shouldn't be examples in default values

Reply:
replaced with {}